env/aarch64

```diff
@@ -161,13 +161,24 @@ export BUILDPERL32='#'
 #export PYTHON3_VERSION=3.5
 #export PYTHON3_PKGVERS=-35
 #export PYTHON3_SUFFIX=m
-for v in {13..9}; do
-	[ -x /usr/bin/python3.$v ] || continue
-	PYTHON3_MINOR=$v
-	export PYTHON3_VERSION=3.$v
-	export PYTHON3_PKGVERS=-3$v
-	break
-done
+if [[ ${MACH} == ${NATIVE_MACH} ]]; then
+	for v in {13..9}; do
+		[ -x /usr/bin/python3.$v ] || continue
+		PYTHON3_MINOR=$v
+		break
+	done
+else
+	PYTHON3_MINOR=$(pkg -R $aarch64_SYSROOT list -H -o version \
+	    runtime/python-3* | cut -d. -f2 | sort -n | tail -1)
+
+	if [ ! -x /usr/bin/python3.$PYTHON3_MINOR ]; then
```

Comment:
I had wondered if we should check for the `Python.h` rather than the interpretter.  I don't know if it makes a difference.

One thing that stands out is that this suggests a bug in illumos-gate/arm64-gate too.  We assume the host and target python are the same in places we don't have to -- #! generation especially.  I wonder if we can break the host/target dependency in more places.  I don't think we can in general.